KarhooUISDK/Screens/BookingScreen/QuoteListMVP/KarhooQuoteListPresenter.swift

```diff
@@ -74,6 +74,15 @@ final class KarhooQuoteListPresenter: QuoteListPresenter {
         default: break
         }
     }
+    
+    private func handleQuotePolling() {
+        let timer = fetchedQuotes!.validity
```

Comment:
Similar to Android should we consider 
       val refreshDelay = if (vehicles.validity >= VALIDITY_DEFAULT_INTERVAL) {
            vehicles.validity.times(VALIDITY_SECONDS_TO_MILLISECONDS_FACTOR)
        } else {
            VALIDITY_DEFAULT_INTERVAL.times(VALIDITY_SECONDS_TO_MILLISECONDS_FACTOR)
        }

Reply:
Not required.

The reason we did that on Android was to make sure the delay was coming back correctly and to handle the conversion of the validity timer to become seconds rather than being treated as milliseconds. So we had to times the validity by 1000 and if the validity was to come back as lower than 5, then it would simply use a 5-second delay.

IOS handles all of that itself. The timer does not require conversion this time and if validity hits 0, it is set to resubscribe to the quote polling and fetch new quotes with the default 5-second refresh until the status is once again marked as completed, in which case it will repeat as expected.